packages/core/src/categories/types.ts

```diff
@@ -1,18 +1,39 @@
 export const CATEGORIES = [
+  // outdated
   'art',
+  'events',
```

Comment:
I think JSDoc notation is more suitable here 
```suggestion
  // @deprecated Outdated
  'art',
   // @deprecated Outdated
  'events',
```

Reply:
It outdated but not exactly called deprecated. Theses values are not available anymore for choose but needed for correct visualize when needed.